templates/taxi/manufacturer_list.html

```diff
@@ -23,6 +25,12 @@ <h1>Manufacturer List
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                <div class="btn-group">
+                  <a class="btn btn-sm btn-outline-secondary" href="{{ manufacturer.id }}/update">Update</a>
```

Comment:
The 'Update' link should use the `{% url %}` template tag to ensure it points to the correct URL. For example: `{% url 'taxi:manufacturer-update' manufacturer.id %}`.